version.go

```diff
@@ -23,4 +23,4 @@ package tchannel
 // VersionInfo identifies the version of the TChannel library.
 // Due to lack of proper package management, this version string will
 // be maintained manually.
-const VersionInfo = "1.0.2"
+const VersionInfo = "1.0.3"
```

Comment:
So once I merge this in, I tag the release with "v1.0.3" and glide supports tags to get specific versions.